waveguide_mode_converter/mode_converter_meep_opt.py

```diff
@@ -63,9 +63,9 @@
 Si = mp.Medium(index=nSi)
 
 design_region_size = mp.Vector3(dx, dy, 0)
-design_region_resolution = int(2 * resolution)
-Nx = int(design_region_size.x * design_region_resolution)
-Ny = int(design_region_size.y * design_region_resolution)
+design_region_resolution = 99.375
```

Comment:
I don't like using a hard-coded value here, rather than making it relative to the `resolution`